src/_reducers/officeLocations.reducer.js

```diff
@@ -27,7 +27,7 @@ export default (state = initialState.officeLocations, action) => {
     case LOAD_LOCATIONS_FAILURE:
       return {
         ...state,
-        error: action.payload,
+        error: action.payload.message,
```

Comment:
What happens when there is a failure and the payload doesn't have a message key? I recommend adding a fallback for this, for example:
```js
error: action.payload.message || 'Oops, something went wrong'